src/realmModels/Observation.js

```diff
@@ -473,7 +473,8 @@ class Observation extends Realm.Object {
       time_observed_at: { type: "string?", mapTo: "timeObservedAt" },
       user: "User?",
       updated_at: "date?",
-      viewed: "bool?"
+      viewed_comments: "bool?",
```

Comment:
does it make more sense to store this on the `Observation` or on the linked `Comment` or `Identification` model?

Reply:
I would keep it in the Observation for now, because the only use case so far is if any one of the comments is not viewed. At the moment it does not matter which one exactly.